app/views/components/v2/summaryCardLPP.scala.html

```diff
@@ -83,11 +83,7 @@
     <header class="app-summary-card__header">
         <h3 class="app-summary-card__title">
             @{
-                if(summaryCard.isAdditionalPenalty) {
-                    messages("summaryCard.header.financialAdditionalPenalty", CurrencyFormatter.parseBigDecimalNoPaddedZeroToFriendlyValue(summaryCard.amountDue))
-                } else {
-                    messages("summaryCard.header.financialNoNumber", CurrencyFormatter.parseBigDecimalNoPaddedZeroToFriendlyValue(summaryCard.amountDue))
-                }
+                messages("summaryCard.header.financialNoNumber", CurrencyFormatter.parseBigDecimalNoPaddedZeroToFriendlyValue(summaryCard.amountDue))
```

Comment:
Could you remove summaryCard.header.financialAdditionalPenalty from messages now?